backoffice/webpack.config.js

```diff
@@ -30,7 +30,19 @@ module.exports = () => {
 			filename: 'main.js'
 		},
 		devServer: {
-			contentBase: path.resolve(__dirname, 'public')
+			contentBase: path.resolve(__dirname, 'public'),
+			proxy: {
+				'/api': {
```

Comment:
@jegj the "/api/api/" is in the dev server just a proxy between localhost:8080/api to localhost:8888 to avoid cors, no need to update the enpoint.